.travis.yml

```diff
@@ -22,6 +22,10 @@ matrix:
   include:
     - php: 7.0
       env: PHPCS=1 DEFAULT=0
+    - php: 7.0
+      env: PHPSTAN=1 DEFAULT=0
+    - php: 7.1
+      env: PHPSTAN=1 DEFAULT=0
```

Comment:
Because PHPStan uses the reflection of current PHP version it runs under, it can possibly find different issues on each one - for example in 7.1.1, third argument to `unpack` function was added so PHPStan can point out whether you are using the parameter on older versions.